src/fp_player.c

```diff
@@ -5,69 +5,400 @@
+                        "Super Smash\0"
+                        "Mega Smash\0"
+                        "Power Smash X\0"
+                        "Power Smash Y\0"
+                        "Deep Focus A\0"
+                        "Super Focus\0"
+                        "Shrink Smash\0"
+                        "Shell Crack\0"
+                        "Kaiden\0"
+                        "D-Down Jump\0"
+                        "Shrink Stomp\0"
+                        "Damage Dodge A\0"
+                        "Earthquake Jump\0"
+                        "Deep Focus B\0"
+                        "Deep Focus C\0"
+                        "HP Plus B\0"
+                        "FP Plus B\0"
+                        "Happy Heart B\0"
+                        "Happy Heart X\0"
+                        "Flower Saver B\0"
+                        "Flower Saver X\0"
+                        "Damage Dodge B\0"
+                        "Damage Dodge X\0"
+                        "Power Plus B\0"
+                        "Power Plus X\0"
+                        "Defend Plus X\0"
+                        "Defend Plus Y\0"
+                        "Happy Flower A\0"
+                        "Happy Flower B\0"
+                        "Happy Flower X\0"
+                        "Group Focus\0"
+                        "Peekaboo\0"
+                        "Attack FXD\0"
+                        "Attack FXB\0"
+                        "Attack FXE\0"
+                        "Attack FXC\0"
+                        "Attack FXF 1\0"
+                        "HP Plus C\0"
+                        "HP Plus X\0"
+                        "HP Plus Y\0"
+                        "FP Plus C\0"
+                        "FP Plus X\0"
+                        "FP Plus Y\0"
+                        "Healthy Healthy\0"
+                        "Attack FXF 2\0"
+                        "Attack FXF 3\0"
+                        "Attack FXF 4\0"
+                        "Attack FXF 5\0"
+                        "Partner Attack\0"
+                        "Heart\0"
+                        "Coin\0"
+                        "Heart Piece\0"
+                        "Star Point\0"
+                        "Full Heal\0"
+                        "Flower\0"
+                        "Star Piece\0"
+                        "Present\0"
+                        "Complete Cake\0"
+                        "Bare Cake\0"
+                        "Empty Cake Pan\0"
+                        "Full Cake Pan\0"
+                        "Empty Mixing Bowl\0"
+                        "Full Mixing Bowl\0"
+                        "Cake With Icing\0"
+                        "Cake With Berries\0"
+                        "Hammer 1 Icon\0"
+                        "Hammer 2 Icon\0"
+                        "Hammer 3 Icon\0"
+                        "Boots 1 Icon\0"
+                        "Boots 2 Icon\0"
+                        "Boots 3 Icon\0"
+                        "Items Icon\0";
 
```

Comment:
since the plan is to get icons working and remove this massive string anyway it prob doesnt matter too much, but maybe want to turn off the formatter so it leaves it as condensed as possible